bigbluebutton-html5/imports/ui/components/presentation/container.jsx

```diff
@@ -22,6 +28,33 @@ export default withTracker(({ podId }) => {
       id: slideId,
     } = currentSlide;
     slidePosition = PresentationAreaService.getSlidePosition(podId, presentationId, slideId);
+    if (PRELOAD_NEXT_SLIDE && !fetchedpresentation[presentationId]) {
+      fetchedpresentation[presentationId] = {
+        fetchedSlide: {},
+      };
+    }
+    const currentSlideNum = currentSlide.num;
+    const presentation = fetchedpresentation[presentationId];
+    if (!presentation.fetchedSlide[currentSlide.num + PRELOAD_NEXT_SLIDE] && canFetch) {
+      const slidesToFetch = Slides.find({
+        podId,
+        presentationId,
+        num: {
+          $in: Array(PRELOAD_NEXT_SLIDE).fill(1).map((v, idx) => currentSlideNum + (idx + 1)),
+        },
+      }).fetch();
+
+      const promiseImageGet = slidesToFetch
+        .filter(s => !fetchedpresentation[presentationId].fetchedSlide[s.num])
+        .map(async (slide) => {
+          if (canFetch) canFetch = false;
```

Comment:
As far as I understood, it seems that this code supports multiple presentations, so the canFetch should consider the id of the presentation. Maybe store it inside the presentation object too (just like you did with fetchedSlide property)?